tornado/test/websocket_test.py

```diff
@@ -260,6 +270,16 @@ def test_path_args(self):
         self.assertEqual(res, 'hello')
 
     @gen_test
+    def test_coroutine(self):
+        ws = yield self.ws_connect('/coroutine')
+        yield ws.write_message('hello1')
+        res = yield ws.read_message()
+        self.assertEqual(res, 'hello1')
+        yield ws.write_message('hello2')
+        res = yield ws.read_message()
+        self.assertEqual(res, 'hello2')
```

Comment:
This test, and test_native_coroutine below, now write two messages and read two responses, instead of one.